podvm-mkosi/mkosi.presets/system/mkosi.conf.d/fedora-debug.conf

```diff
@@ -0,0 +1,17 @@
+[Match]
+Distribution=fedora
+# This is a little hack to define different image types in a mkosi config.
+# There is also imageId, but it renames the output, which is not what we want.
+PathExists=../../resources/buildDebugImage
+
+[Content]
+Autologin=true
+KernelCommandLine=rd.shell
+KernelCommandLine=systemd.setenv=SYSTEMD_SULOGIN_FORCE=1
+SkeletonTrees=../../mkosi.skeleton-debug
+Packages=
+    nano
+    vim
+    strace
+    openssh-server
+    file
```

Comment:
Do we need to include dnf/yum or something to be able to install additional commands? I found myself often requiring to install additional debugging utilities on podvms.

Reply:
Doesn't make much sense, as the disk will be read-only in a few PRs.

Reply:
I see, I feel we need a facility for that, though - maybe an overlayroot backed by tmpfs? workarounds like privileged sidecar wouldn't help if we have agent problems.

Reply:
If we fail to start a containers via kata + remote-hypervisor, we'd need something to start debug containers locally, after ssh'ing into the instance. It doesn't need to be super straightforward, or even builtin. but we should document a way to do this. not having the option to access additional tooling would be prohibitive for debugging peerpod issues from my experience.

Reply:
fwiw, for one of my debugging scenarios, I did add dnf during the image build to make it easier to install packages during debugging.